nethttp/status-code-tracker-old.go

```diff
@@ -9,14 +9,24 @@ import (
 
 type statusCodeTracker struct {
```

Comment:
I recommend not touching  this file, which is for Go 1.7. in fact, I would vote to delete it altogether, the typical contract for Go libs  is to support the last to versions of Go. 1.7 is very old.

Reply:
I think it is better to drop support for Go 1.7 in another separated PR. The code base now includes the support for Go 1.7, then the bugfix in current PR should keep the support for Go 1.7.